react-carousel/src/state/atoms/slideAtoms.js

```diff
@@ -0,0 +1,21 @@
+import { atom } from 'recoil';
```

Comment:
is recoil an alternative for redux?

if so why recoil?

Reply:
IMO it requires less boilerplate code to start using it, it's simpler to understand and manage in small projects/libs like this one

Reply:
ok